dot/parachain/candidate-validation/host.go

```diff
@@ -0,0 +1,79 @@
+package candidatevalidation
+
+import (
+	"fmt"
+
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	"github.com/ChainSafe/gossamer/internal/log"
+	"github.com/ChainSafe/gossamer/pkg/scale"
+)
+
+var logger = log.NewFromGlobal(log.AddContext("pkg", "pvf"), log.SetLevel(log.Debug))
+
+type Host struct {
```

Comment:
any specific reason why this is public? Seems to be only referenced in the package.

Reply:
no reason, I've updated this to be private.